app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema")
```

Comment:
it's better when your db name has one of db file extensions ( cinema.db3 or cinema.sqlite for example)